ios/Classes/AblyFlutterPlugin.m

```diff
@@ -685,8 +690,10 @@ - (void)application:(UIApplication *)application didFailToRegisterForRemoteNotif
 }
 
 - (BOOL)application:(UIApplication *)application didReceiveRemoteNotification:(NSDictionary *)userInfo fetchCompletionHandler:(void (^)(UIBackgroundFetchResult))completionHandler {
-    // This needs to return NO allow the message to arrive to the users application AppDelegate.
-    return NO;
+    [_pushNotificationEventHandlers application:application didReceiveRemoteNotification:userInfo fetchCompletionHandler:completionHandler];
+    // TODO try this, to give other plugins and AppDelegate a chance to receive the remote notification as well. If NO/ false is returned, the user's didReceiveRemoteNotification delegate method will be called
```

Comment:
Oh this is one I left recently because it requires some thought. In firebase_messaging, if the APNs message has `userInfo[@"gcm.message_id"]`, then it returns `YES`, and the users app delegate method **won't** be informed of the remote notification. However, we want to allow customers to implement app delegate methods (handle messages natively) if they choose to, even if our plugin is installed. So we would have to return `NO`, that our plugin did not handle the message. 

I've done that here: d562fd4c1ef9f257412a16033acb17ec928fabc2